Tests/AutoCheckTools.h

```diff
@@ -13,6 +13,22 @@
 
 #include "MathLib/Point3d.h"
 
+
+namespace std{
```

Comment:
I thought the namespace `std` is exclusively for standard template library. Why is it neccessary to put the operator in the namespace?